backend/open_webui/retrieval/web/brave.py

```
@@ -12,31 +12,41 @@
         "Accept": "application/json",
         "Accept-Encoding": "gzip",
         "X-Subscription-Token": api_key,
+        "Content-Type": "application/json",
     }
-    params = {"q": query, "count": count}
 
-    response = requests.get(url, headers=headers, params=params)
+    # Prepare the request body for chat completions
+    payload = {"stream": "false", "messages": [{"role": "user", "content": query}]}
+
+    response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)